langchain4j-core/src/test/java/dev/langchain4j/agent/tool/ToolExecutionRequestUtilTest.java

```diff
@@ -12,13 +12,14 @@ public void test_argument() {
                 .arguments("{\"foo\":\"bar\", \"qux\": 12}")
                 .build();
 
+        // TODO: Langchain4j: is ok to have now either long or double ? It fixes other issues.
```

Comment:
what kind of issues?

Reply:
https://github.com/langchain4j/langchain4j/pull/1353/files#diff-d8935c746a9b58544db4c7e3a9303e693a4b2353001c3af5038db4586e3324f8L425-L479